deploy/charts/cert-manager/templates/deployment.yaml

```diff
@@ -236,4 +241,4 @@ spec:
       {{- end }}
       {{- with .Values.hostAliases }}
       hostAliases: {{ toYaml . | nindent 8 }}
-      {{- end }}
\ No newline at end of file
+      {{- end }}
```

Comment:
nit: probably best to leave the newline at the end of the file. I think that is the convention, always have 1 newline.